client/js/styleguide/palette.js

```diff
@@ -24,10 +24,13 @@
   })
 
   pairArray.forEach((pair) => {
-    palette_element.innerHTML += `
-    <div class="styleguide__hex" style="background: ${pair.hex};">
-      <span class="styleguide__color">${pair.name}: <code class="styleguide__hex-code">${pair.hex}</code></span>
-    </div>`
+    // ignore 'currentColor', 'transparent' and 'inherit'
+    if (pair.hex.indexOf('#') !== -1) {
+      palette_element.innerHTML += `
```

Comment:
I think if we do it in the templating, we would be required to ensure it was up-to-date manually, whereas with this elegant/inelegant (you decide) hack, we can always ensure what displays in the palette is exactly what's laid out in the scss.